internal/services/springcloud/spring_cloud_java_deployment_resource.go

```diff
@@ -136,12 +136,9 @@ func resourceSpringCloudJavaDeploymentUpdate(d *pluginsdk.ResourceData, meta int
 		existing.Sku.Capacity = utils.Int32(int32(d.Get("instance_count").(int)))
 	}
 
-	if features.ThreePointOhBeta() && d.HasChange("cpu") {
-		existing.Properties.DeploymentSettings.CPU = utils.Int32(int32(d.Get("cpu").(int)))
-
-		// "cpu" within "quota" that takes precedence of deprecated "cpu" should be ignored in this situation where users explicitly update the deprecated "cpu" that conflicts with "cpu" within "quota"
+	if d.HasChange("cpu") {
```

Comment:
this would be a breaking change without the feature flag

Reply:
I think the feature flag is not necessary, since `cpu` will only be in schema when it's not 3.0.